docs/reference/transform/apis/schedule-now-transform.asciidoc

```diff
@@ -24,10 +24,10 @@ in the `transform_admin` built-in role.
 [schedule-now-transform-desc]]
 == {api-description-title}
 
-If you _schedule_now a {transform}, it will process the new data instantly,
+If you `_schedule_now` a {transform}, it will process the new data instantly,
```

Comment:
IMO the use of the endpoint name as a verb is unwieldy. I think it's simpler like this:
```suggestion
When you run this API, the {transform} processes the new data instantly,
```